sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module

```diff
@@ -123,6 +123,9 @@ function reol_lazy_load_images_theme_registry_alter(&$theme_registry) {
  */
 function theme_reol_base_image($variables) {
   $attributes = $variables['attributes'];
+  // hide all <img from screen readers
```

Comment:
According to the Drupal coding standards
* Inline comments must start with a capital letter
* Inline comments must end in full-stops, exclamation marks, question marks, colons, or closing parentheses

Reply:
nu spørger jeg, selvom jeg godt ved du ikke har skrevet de guidelines, hvornår exclamation marks?